src/RobaseService/init.lua

```diff
@@ -18,31 +14,22 @@ local function fixTrailingSlashes(url)
     return url
 end
 
-function RobaseService.new(baseUrl)
+function RobaseService.new(baseUrl, token)
```

Comment:
I can't merge a PR with this kind of breaking change, either I wait till 3.x to pull this or we allow this as an optional parameter?